3.9/analyzers.md

```diff
@@ -802,7 +803,41 @@ lower-case and base characters) and then discards the stopwords `and` and `the`:
     @endDocuBlock analyzerPipelineStopwords
 {% endarangoshexample %}
 {% include arangoshexample.html id=examplevar script=script result=result %}
-{% endcomment %}
+
+### `collation`
+
+<small>Introduced in: v3.9.0</small>
+
+An Analyzer capable of conveting the input into a set of language-specific tokens.
+
+The *properties* allowed for this Analyzer are an object with the following
+attributes:
+
+- `locale` (string): a locale in the format
+  `language[_COUNTRY][.encoding][@variant]` (square brackets denote optional
+  parts), e.g. `"de.utf-8"` or `"en_US.utf-8"`. Only UTF-8 encoding is
+  meaningful in ArangoDB. Also see [Supported Languages](#supported-languages).
+
+### `segmentation`
+
+An Analyzer capable of breaking up the input text into tokens in language
+agnostic manner as per [Unicode Standard Annex #29](https://unicode.org/reports/tr29) 
+while also optionally applying case conversion.
+
+The *properties* allowed for this Analyzer are an object with the following
+attributes:
+
+- `break` (string, _optional_):
+  - `"all"` return all tokens
+  - `"alpha"` return tokens composed of alphanumeric characters (default)
+  - `"graphic"`return tokens composed of printable characters
+- `case` (string, _optional_):
+  - `"lower"` to convert to all lower-case characters (default)
+  - `"upper"` to convert to all upper-case characters
+  - `"none"` to not change character case
+
+
+<small>Introduced in: v3.9.0</small>
```

Comment:
Suggest to move it to the beginning of the analyzer description as we always  do it like this for other analyzers